src/mscorlib/model.xml

```diff
@@ -3693,21 +3733,26 @@
       <Member Name="#ctor" />
       <Member Name="get_CultureInfo" />
       <Member Name="get_CultureName" />
+      <Member Name="get_CodeBase" />
```

Comment:
what I meant is that you did add the getter and setter for CodeBase, but you didn't add it as a MemberType=Property. Again, I'm not sure if that is required or not, but if you see looks like we are doing that for other properties in this same type.